com.unity.uiwidgets/Runtime/rendering/list_wheel_viewport.cs

```diff
@@ -531,7 +531,7 @@ public class RenderListWheelViewport : ContainerRenderObjectMixinRenderBox<Rende
             var radius = this.size.height * this._diameterRatio / 2.0f;
             var deltaY = radius * Mathf.Sin(angle);
 
-            Matrix3 transform = Matrix3.I();
+            Matrix4 transform = new Matrix4().identity();
```

Comment:
here is the place where we should use Matrix4 instead of Matrix3. I believe it will be very helpful if you can check whether this widget looks correctly after these changes !